AirCasting/SessionViews/SessionHeaderView.swift

```diff
@@ -11,6 +11,7 @@ struct SessionHeaderView: View {
     let action: () -> Void
     let isExpandButtonNeeded: Bool
     var isSensorTypeNeeded: Bool = true
+    var isMenuNeeded = true
```

Comment:
It is never used as I can see. Can we delete it?

Reply:
It is necessary and used in line 34

Reply:
but it will always be true, won't it?